src/shared/charts/tradeChartData.ts

```diff
@@ -26,43 +31,61 @@ export function getTradeEntries(dataset: PairHistory, filteredTrades: Trade[]) {
   // 4: color
   // 5: label
   // 6: tooltip
-
   for (let i = 0, len = filteredTrades.length; i < len; i += 1) {
     const trade: Trade = filteredTrades[i];
     if (
       trade.open_timestamp >= dataset.data_start_ts &&
```

Comment:
Does this stop trade adjustments showing where the trade entry was prior to dataframe timerange?

Also ref issue #1093